concourse/steps/component_descriptor.mako

```diff
@@ -1,8 +1,9 @@
 <%def
-  name="component_descriptor_step(job_step, job_variant, output_image_descriptors, indent)",
+  name="component_descriptor_step(job_step, job_variant, output_image_descriptors, default_github_config, indent)",
```

Comment:
are you sure it is necessary to pass-in the default_gh_cfg? rather calculate if from hostname (ccc.github), if even required in the first place.